sor/src/test/java/com/bazaarvoice/emodb/sor/db/DeltaBlockingTest.java

```diff
@@ -126,9 +128,11 @@ public void testFragmentedDelta() throws IOException {
             rows.add(new TestRow(i, secondDeltaUUID, secondDeltaBlocks.get(i)));
         }
 
-        Iterator<ByteBuffer> iterator = new ListDeltaIterator(rows.iterator(), false, _prefixLength);
+        Iterator<DeltaIterator.BlockedDelta> iterator = new ListDeltaIterator(rows.iterator(), false, _prefixLength);
         assertEquals(iterator.hasNext(), true);
-        assertEquals("000AD3:[]:0:{..,\"name\":\"bobåååååຄຄຄຄຄຄຄຄຄຄ\"}", StringSerializer.get().fromByteBuffer(iterator.next()));
+        DeltaIterator.BlockedDelta secondBlockedDelta = iterator.next();
+        assertEquals("000AD3:[]:0:{..,\"name\":\"bobåååååຄຄຄຄຄຄຄຄຄຄ\"}", StringSerializer.get().fromByteBuffer(secondBlockedDelta.getContent()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "000AD3:[]:0:{..,\"name\":\"bobåååååຄຄຄຄຄຄຄຄຄຄ\"}" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)